appstudio-utils/util-scripts/check-ec-policy.sh

```diff
@@ -1,20 +1,25 @@
 #!/usr/bin/bash
-set -euo pipefail
+set -xeuo pipefail
 
 EC_WORK_DIR=${EC_WORK_DIR:-/tmp/ecwork}
 
 DATA_DIR=$EC_WORK_DIR/data
+INPUT_DIR=$EC_WORK_DIR/input
 POLICY_DIR=$EC_WORK_DIR/policies
 
 FORMAT=${1:-pretty}
 
-OPA_QUERY=${OPA_QUERY:-data.hacbs.contract.main.deny}
+OPA_QUERY=${OPA_QUERY:-hacbs.contract.test}
 
 [[ ! -d $DATA_DIR ]] && echo "Data dir $DATA_DIR not found!" && exit 1
+[[ ! -d $INPUT_DIR ]] && echo "Input dir $INPUT_DIR not found!" && exit 1
 [[ ! -d $POLICY_DIR ]] && echo "Policy dir $POLICY_DIR dir not found!" && exit 1
 
-opa eval \
-  --data $DATA_DIR \
-  --data $POLICY_DIR \
-  --format $FORMAT \
-  $OPA_QUERY
+INPUT_FILES=$( find ${INPUT_DIR} -name input.json )
```

Comment:
Do we need the fine here? I think this should work similar:

    INPUT_FILE="$INPUT_DIR/input.json"